Python/PRP/Surface/pyMipmap.cpp

```diff
@@ -142,7 +142,12 @@ PY_METHOD_VA(Mipmap, setLevel,
         PyErr_SetString(PyExc_TypeError, "setLevel expects int, binary string");
         return NULL;
     }
-    self->fThis->setLevelData(level, (const void*)data, dataSize);
+    try {
+        self->fThis->setLevelData(level, (const void*)data, dataSize);
+    } catch (hsBadParamException& ex) {
```

Comment:
Should catch by `const` reference

Reply:
Heh, I had it this way... Then noticed the rest of the file was catching by non-`const` reference. Fixed.